homeassistant/components/atag/.translations/en.json

```diff
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "title": "Atag",
```

Comment:
```suggestion
    "title": "Atag",
    "config": {
```

title is now top level

Reply:
Caught that one during the pre-commit :)